projects/Mallard/package.json

```diff
@@ -10,7 +10,7 @@
         "test": "jest --coverage",
         "test:watch": "jest --watch",
         "shake-android": "echo '👋 📱' && adb shell input keyevent 82 && echo '🙌📲🙌'",
-        "wifi-android": "yarn pre-run && ip=$(adb shell ifconfig wlan0  | awk '/inet addr/{print substr($2,6)}') && adb tcpip 5555 && adb connect $ip:5555",
+        "wifi-android": "ip=$(adb shell ifconfig wlan0  | awk '/inet addr/{print substr($2,6)}') && adb tcpip 5555 && adb connect $ip:5555",
```

Comment:
why dont we wanna pre-run?

Reply:
because `wifi-android` just connects to your device over wifi, you still need to run `run-android`